cfgov/jinja2/v1/_includes/organisms/filter-filterable-list.html

```diff
@@ -0,0 +1,159 @@
+
+    {{ _render_form_wrapper(inputs) }}
+
+    <div class="form-actions form-actions__left-on-med">
+        <input class="btn form-actions_item"
+               type="submit"
+               value="Apply Filters">
+        <input class="btn btn__warning btn__link form-actions_item js-form_clear"
+               type="button"
+               value="Clear filters"
+               data-js-hook="form-clear">
+    </div>
+
+</form>
+{% endmacro %}
+
+{# A helper macro for filter_filterable_list.render()
+   ========================================================================== #}
+{% macro _render_form_wrapper(inputs) -%}
+
+    {% for input in inputs recursive %}
+        <div class="form-l_col
+                    form-l_col-{{ input.cols if input.cols else '1' }}">
+            <div class="form-group">
+
+                {% if input.label %}
+                <label class="form-label-header">
+                    {{ input.label }}
+                </label>
+                {% endif %}
+
+                {% if input.type == 'form' %}
+                    <div class="form-l">
+                        {{ loop(input.children) }}
+                    </div>
+                {% elif input.type == 'email'
+                   or input.type == 'number'
+                   or input.type == 'password'
+                   or input.type == 'tel'
+                   or input.type == 'text'
+                   or input.type == 'url' %}
+
+                    <input type="{{ input.type if input.type else 'text' }}"
+                           value="{{ input.attrs.value if input.attrs.value else '' }}"
+                           placeholder="{{ input.attrs.placeholder if input.attrs.placeholder else '' }}">
+                {% elif input.type == 'checkbox' %}
+                    {% for checkbox in input.attrs -%}
+                        <label class="form-group_item">
+                            {{ _filter_checkbox(checkbox.name, checkbox.value, checkbox.label) | safe }}
+                        </label>
+                    {% endfor %}
+                {% endif %}
+            </div>
+        </div>
+    {% endfor %}
+{%- endmacro %}
+
+{# A helper macro for filter_filterable_list.render()
+   ========================================================================== #}
+{% macro _filter_checkbox(field, value, label) %}
+    <input class="custom-input"
+           type="checkbox"
+           name="filter_{{ field }}"
+           value="{{ value }}"
+           {%- if is_filter_selected(field, value) %}
+           checked
+           {%- endif -%}
+           >
+        {{ label if label else value }}
+{% endmacro %}
+
+{# A helper macro for filter_filterable_list.render()
+   ========================================================================== #}
+{% macro _filter_option(field, value) %}
+    <option value="{{ value }}"
+        {%- if is_filter_selected(field, value) %}
```

Comment:
Also, added `select` option as I realized that wasn't actually supported before:

![screen shot 2015-12-04 at 11 12 34 am](https://cloud.githubusercontent.com/assets/704760/11594418/ef59a568-9a77-11e5-978a-d755d4ef21ee.png)